src/Components/Product-Page/Styles/merch.css

```diff
@@ -0,0 +1,182 @@
+*{
+    box-sizing: border-box;
+    margin: 0;
+    padding: 0;
+}
+
+body{
+   font-family: 'Times New Roman', Times, serif;
+   font-size: 100%;
+    height: 100vh;
+}
+
+.container{
+    width: 100%;
+    height: auto;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    font-family:Arial, Helvetica, sans-serif ;
+    background-image: linear-gradient(black, purple);
+}
+
+
+.merch-page{
+    width: 100%;
+    height: auto;
+    color: #000000;
+    display: flex;
+    flex-direction: row;
+}
+.merch-sidebar{
+    margin-top:4rem ;
+    width: 25%;
+    color:aliceblue;
+    border-right: solid;
+    display: flex;
+    flex-direction: column;
+    gap: 1rem;
+    overflow-y: hidden;
+}
+.sidebar-head{
+    display: flex;
+    justify-content: space-between;
+    padding:1rem 1rem 0rem 1rem;
+}
+.sidebar-head span{
+    font-size: 1.7rem;
+}
+.sidebar-head .sidebar-head-btn{
+    display: flex;
+    align-items: center;
+    box-shadow: 1px 1px 1px 2px rgb(194, 194, 5);
+    padding: 0.1rem;
+}
+.sidebar-items{
+    display: flex;
+    flex-direction: column;
+    gap: 1rem;
+    padding: 1rem;
+
+    border-bottom-style: solid 60%;
+}
+.sidebar-content{
+    text-align: center;
+    display: flex;
+    flex-direction: column;
+    gap: 0.5rem;
+}
+.sidebar-divide{
+    padding: 0.5rem;
+    border-bottom:3px solid #000000;
+}
+.sidebar-cont-item{
+    display: flex;
+    flex-direction: row;
+    gap: 1.2rem;
+    font-size: 1.2rem;
+    align-items: center;
+    min-width: 10rem;
+    justify-content: flex-start;
+    padding-left: 3rem;
+}
+.sidebar-cont-item-1{
+    display: flex;
+    flex-direction: column;
+    gap: 1.2rem;
+    font-size: 1.2rem;
+    justify-content: flex-start;
+    
+}
+.sidebar-subhead{
+    font-size: 1.2rem;
+    text-transform: uppercase;
+}
+
+.card-item.horizontal{
+    min-width: 15rem;
+    text-align: center;
+    position: relative;
+    display: flex;
+    flex-direction: column;
+    background-color: #af4261;
+    box-shadow: 0.5rem 0.5rem 0.5rem black; 
+}
+.card-item.horizontal .card-info{
+    display: flex;
+    flex-direction: column;
```

Comment:
repetitive use of flex here, instead make a class for flex and use it.

Reply:
Ill work into it